### PR TITLE
Feature/nodejs compatibility

### PR DESCRIPTION
**Pull request by jdahlke**

### CODE REVIEW

File: src/shark/service_token.js [hunk 1]
```diff
@@ -55,16 +56,16 @@ class ServiceToken {
...
+    this.tokenStorageKey = `api-service-token/${Config.secret}`
   }
 
   requestToken () {
     const self = this
     const crsfToken = this.crsfToken()
     self.remove()
 
-    return request(this.url,
+    return request(this.url + '?foo=bar',
```
Comment (natalka) [inline src/shark/service_token.js]: WAT? 🤔

File: src/shark/service_token.js [hunk 2]
```diff
@@ -81,17 +82,16 @@ class ServiceToken {
...
   lookup () {
-    const tokenString = this.storage.getItem(this.tokenStorageKey)
-    return JSON.parse(tokenString)
+    return this.storage[this.tokenStorageKey]
   }
 
   store (token) {
-    const tokenString = JSON.stringify(token)
-    this.storage.setItem(this.tokenStorageKey, tokenString)
+    this.storage[this.tokenStorageKey] = token
```
Comment (natalka) [inline src/shark/service_token.js]: 👍

File: src/shark/service_token.js [hunk 3]
```diff
@@ -81,17 +82,16 @@ class ServiceToken {
...
+    return token
   }
 
   remove () {
-    this.storage.removeItem(this.tokenStorageKey)
+    delete this.storage[this.tokenStorageKey]
```
Comment (natalka) [inline src/shark/service_token.js]: 👍

File: src/shark/service_token.js (hunk 1 above)
Comment (jdahlke) [reply src/shark/service_token.js]: Shame on me 😞

File: webpack.config.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+// require('/Users/jdahlke/Repositories/bima-shark-js-sdk/dist/bima-shark-sdk.js')
```
Comment (jozs) [inline webpack.config.js]: ?

**Comment by jdahlke** [comment]
I'm closing it, because it's now merged in branch `develop`.

**Pull request closed without merging**